### ISSUE TITLE
Migrate world data from Firestore to Airtable 

### ISSUE THREAD

**Issue by shayantist** [reach4help/reach4help#1375]
Since we've already been syncing data from Airtable to Firestore (and using Airtable as a comprehensive solution for entering and updating data), migrating data backwards from Firestore to Airtable is our next step. 

Each marker in Firestore is a JSON object that looks something like this: 
![image](https://user-images.githubusercontent.com/29135917/121862052-4415f480-cd1c-11eb-82f4-fcc7906f1c88.png)

Basically, we have to flatten this (and all the other markers) out to a spreadsheet/CSV and then import that into Airtable. 

[This could be a good start.](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.json_normalize.html)
